index.html

```diff
@@ -26,5 +26,183 @@ <h1>José Fernández-Caminero</h1>
             <img src="resources/image_geometry_2.svg" alt="Geometrical decoration 2">
         </footer>
     </section>
+
+    <section>
+        <h1>Projects</h1>
+
+        <section class="box1">
```

Comment:
Kindly, I recommend you to use SECTION tags for each theme of your page, inside these, you could only use DIV, H, P, etc, but using SECTION inside SECTION is not well applied.